pastis/launchers/run_luvex_efields.py

```diff
@@ -1,36 +1,37 @@
 import os
-import numpy as np
 from astropy.io import fits
+import numpy as np
```

Comment:
I am not sure the logic behind flipping they numpy imports.

Reply:
I sorted them alphabetically.